src/remote-config/remote-config-api-client.ts

```diff
@@ -118,6 +125,7 @@ export class RemoteConfigApiClient {
         return {
           conditions: resp.data.conditions,
           parameters: resp.data.parameters,
+          parameterGroups: resp.data.parameterGroups,
```

Comment:
Is this always present on the response?

Reply:
If `parameterGroups` is not present then we assign it to an empty object. Same for `conditions` (empty array) and `parameters`. Good catch for tests! Added more unit tests to cover these. Thanks!

Reply:
Also run the code coverage tools and see if anything else has slipped through the cracks.